packages/zone/src/virtual.js

```diff
@@ -46,6 +47,8 @@ const defineVirtualExo = (
 export const detachedVirtualStores = Far('virtualStores', {
   detached: () => detachedVirtualStores,
   isStorable: canBeDurable,
+  once: alwaysOnce,
```

Comment:
Should a virtual once enforce that the value produced by the maker is virtual compatible?

Reply:
What Passables are not virtual compatible?

Reply:
Where is `Passable` enforced? that's my question.

Reply:
Got it. I am not sure it is consistently enforced. So yes, the virtual-compatible test could just be a Passable test.